### PR TITLE
Warning on Battery Buzzer Disconnect

### PR DESCRIPTION
**Pull request by kevinmasd**
Label for battery detects if incoming voltage is less than 1, given we agreed with runtime that errors should give out 0 Volts.

Cleaned up fake_runtime for all displayable sensors. Did not know that UPDATE_MOTOR was borked for now.

### CODE REVIEW

**Comment by AllanYangZhou** [comment]
Sorry, UPDATE_MOTOR is borked?

Edit: Although I see updateMotor is never actually used anywhere.

File: dawn/js/components/StatusLabel.js [hunk 1]
```diff
@@ -13,11 +13,14 @@ var StatusLabel = React.createClass({
...
-	labelText = 'Runtime Error';
+	       labelStyle = 'danger';
+	       labelText = 'Runtime Error';
+      } else if (this.props.battery < 1) {
+          labelStyle = 'warning';
+          labelText = 'Battery Issue'
       } else {
-	labelStyle = 'success';
-	labelText = `Connected. Battery: ${numeral(this.props.battery).format('0.00')} V`;
+          labelStyle = 'success';
```
Comment (AllanYangZhou) [inline dawn/js/components/StatusLabel.js]: Was there a reason for increasing the indent here?

File: dawn/js/components/StatusLabel.js [hunk 2]
```diff
@@ -13,11 +13,14 @@ var StatusLabel = React.createClass({
     let labelText = 'Disconnected';
     if (this.props.connectionStatus) {
       if (!this.props.runtimeStatus) {
-	labelStyle = 'danger';
-	labelText = 'Runtime Error';
+	       labelStyle = 'danger';
+	       labelText = 'Runtime Error';
+      } else if (this.props.battery < 1) {
+          labelStyle = 'warning';
```
Comment (AllanYangZhou) [inline dawn/js/components/StatusLabel.js]: Should be 2 spaces indent

**Comment by kevinmasd** [comment]
Sorry; used tabbing, but does that really need another commit?

Noticed that `fake_runtime.py` has always used `UPDATE_PERIPHERAL` since it was created. I tried to convert a motor to `UPDATE_MOTOR`, but instead Dawn wouldn't show its data. Maybe its a limitation of using a fake Runtime or perhaps wrong configuration somewhere.

**Comment by AllanYangZhou** [comment]
Yeah, you should fix the indentation w/ another commit. You can try using squashing the commit if you don't want to pollute your history.

I see what you mean about the UPDATE_MOTOR. Motors and peripherals used to be separate, but now motors are just considered one type of peripheral. However, some of Dawn's code has not been updated to reflect this (I mean, it still works, it's just confusing).

**Comment by kevinmasd** [comment]
Done; same named commit was just a coincidence.

**Pull request merged**